forwards_paper.tex

```diff
@@ -512,11 +512,11 @@ \subsection*{Recording the pedigree in forwards time}
 adds a new node to the table $\Nt$ with time $t$
 and returns the ID of this new node.
 Similarly, the function $\taddrow{\Et}{\ell, r, p, c}$
-adds a new edge $(\ell\mbox{eft}, r\mathrm{ight}, p\mathrm{arent}, c\mathrm{hild})$ to the edge table $\Et$.
```

Comment:
Thanks --- I started editing these and got distracted by something else! My reasoning for changing mathrm BTW is I was worried it would use math-mode spacing unless explicitly escaped.